backend/data/program-credentials/merge-credentials.py

```diff
@@ -160,6 +160,28 @@ def label_credential_type(row: pd.Series):
     if (row['LEADTOINDUSTRYCREDENTIAL'] == "1"):
         return CredentialType.Certification
 
+    # TODO: Track Down Lookup Values for Provider TYPEIDs.
+    # However, data analysis of our current dataset showed looking at the degree awarded field
+    # for providers with these TYPEIDs where true postitives for that credential type while
+    # providers with other TYPEIDs where false postitives for degree credential types
+    TYPEIDS_FOR_DEGREE_GRANTING_PROVIDERS = { "3", "4" }
+
+    # Label Associate Degree awarded from degree granting provider
+    if row['DEGREEAWARDED'] == "200" and row['PROVIDERS_TYPEID'] in TYPEIDS_FOR_DEGREE_GRANTING_PROVIDERS:
```

Comment:
Would it be useful to have a lookup mapping for the `DEGREEAWARDED` values?

Reply:
Do you mean like a dictionary in the python code itself? I think that's a good idea. Something that maps the DEGREEAWARDED to the particular credential type for it. Will add.

The one thing I was conflicted on was whether to use the ID here (the 200) or the DEGREEAWARDEDNAME from the lookup file (would be "Associate's Degree" here). The 200 I think is less susceptible to sudden unexpected change, while the DEGREEAWARDNAME is more readable. I think adding in the dictionary mapping would help to allow me to keep using the ID value here for safety while improving the readability more than a bit.